apps/electron/e2e/fixtures/orchestration.fixture.ts

```
@@ -0,0 +1,256 @@
+
+  for (const session of [orchestrator, childA, childB]) {
+    const dir = path.join(sessionsDir, session.id)
+    mkdirSync(dir, { recursive: true })
+    writeSessionJsonl(path.join(dir, 'session.jsonl'), session)
+  }
+
+  return testDataDir
+}
+
+export const test = base.extend<OrchestrationFixtures>({
+  electronApp: async ({}, use) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)